docs/chain/collator.md

```diff
@@ -1,136 +1,116 @@
+#### Peregrine and Spiritnet chain specs
 
-The `--chain` parameter decides which blockchain we want to execute.
+The `--chain` parameter decides which blockchain the KILT collator will join.
 We need to pass this parameter to the parachain and the relaychain, since both chains are separate blockchains.
-The parachain accepts an additional runtime parameter that selects which runtime is executed and decides which execution environment is used.
+The KILT parachain accepts an additional runtime parameter to select which runtime is executed.
 This can either be `peregrine` or `spiritnet`.
-For now this parameter isn't required, since both runtimes are very similar and can run in the same environment, but it is safer to provided it already now.
+For now this parameter is not required, since both runtimes are very similar and can run in the same environment, but it is safer to provide it already now.
 
 #### Where are all the files stored?
 
 The `--base-path` parameter configures where all the persistent files are written.
-By default the session keys will also be stored int he *base path*, but we recommend to separate them from the other files.
-This makes sure that the keyfiles are not accidentally lost or published when you backup or restore your blockchain database.
-You can configure where to store the session keys using `--keystore-path`.
-Since you only use session keys for the parachain, there is no need to add this to the relaychain part of the command.
+By default the session keys will also be stored in the *base path*, but we recommend to separate them from the other files.
+This makes sure that the keyfiles are not accidentally lost or published when the blockchain database is backed up or restored.
+You can configure where to store the session keys using the `--keystore-path` option.
+Since the collator will collate only for the parachain, there is no need to add this to the relaychain part of the command.
 
-### Acquire the executable
+### Run a Peregrine collator
 
 <Tabs
   groupId="exec-strategy"
   defaultValue="Docker"
   values={[
-    {label: 'Native', value: 'Native'},
+    {label: 'Binary', value: 'Binary'},
     {label: 'Docker', value: 'Docker'},
   ]}>
-<TabItem value="Native">
+<TabItem value="Binary">
 
-We recommend following the [repository](https://github.com/KILTprotocol/mashnet-node).
-Below is the command to connect as a Collator in the root directory of the mashnet-node repository.
+We recommend following the instructions in the [KILT chain repository](https://github.com/KILTprotocol/mashnet-node).
+Below is the command to build the KILT collator executable for the Peregrine network. The command must be run from the root directory of the repository after it has been cloned.
 
 :::info
 
-Do not use master branch to compile the build.
-Either use `develop` or the following tag `1.1.1`.
+We discourage to use the `develop` branch to build the executable. Instead, the last released version should be used.
```

Comment:
Not sure about that. It only matters if they use the native runtime.

Reply:
So the question is: what message to we want to convey here? That `master` should not be used, like it was before, or that it is, on the contrary, preferable even though not required?

Reply:
Ah right! I didn't notice the part about master. Yes so they could use both. But we might want to tell them to only use master.

Reply:
So `master` or the latest release?

Reply:
master i would say